### DISCUSSION TITLE
XTTS2 inference with speaker_idx argument, without source_wav

### DISCUSSION THREAD

**Discussion by pivolan** [idiap/coqui-ai-TTS#138] [Q&A] [answered]
Is it real to inference this model by using just one of the inner speakers, to prevent voice cloning? When I try to find speakers list by args --list_speaker_idxs, it just send me list of embeddings, but not speakers list.

**Comment by pivolan** [comment]
[ 1.4573e-02]]])}, 'Damjan Chapman': {'gpt_cond_latent': tensor([[[-0.1542,  1.0372,  0.0600,  ..., -0.4648, -0.6890,  2.2647],
         [-0.3850,  0.1696,  1.0869,  ..., -0.3312,  0.7753,  1.5265],
         [-0.4134,  0.6549, -1.0348,  ..., -0.2458, -0.6853,  1.4418],
         ...,
         [-0.4208,  0.5191, -0.6756,  ..., -0.2182, -0.5243,  1.1794],
         [-0.6379,  1.5274,  0.2272,  ..., -1.5168, -0.2494,  1.1695],
         [ 0.3209, -0.0421,  0.7555,  ...,  0.8700, -0.3047,  2.2061]]]), 'speaker_embedding': tensor([[[-8.7840e-03],
         [ 1.7883e-02],
         
         this is an example of output

**Reply by eginhard** [reply, marked as answer]
That's indeed not very useful, thanks for flagging. I opened #139 to track improving the output here.

**Comment by pivolan** [comment]
just grep 'gpt_cond_latent' and got this list of speakers.
Claribel Dervla, Daisy Studious, Gracie Wise, Tammie Ema, Alison Dietlinde, Ana Florence, Annmarie Nele, Asya Anara, Brenda Stern, Gitta Nikolina, Henriette Usha, Sofia Hellen, Tammy Grit, Tanja Adelina, Vjollca Johnnie, Andrew Chipper, Badr Odhiambo, Dionisio Schuyler, Royston Min, Viktor Eka, Abrahan Mack, Adde Michal, Baldur Sanjin, Craig Gutsy, Damien Black, Gilberto Mathias, Ilkin Urbano, Kazuhiko Atallah, Ludvig Milivoj, Suad Qasim, Torcull Diarmuid, Viktor Menelaos, Zacharie Aimilios, Nova Hogarth, Maja Ruoho, Uta Obando, Lidiya Szekeres, Chandra MacFarland, Szofi Granger, Camilla Holmström, Lilya Stainthorpe, Zofija Kendrick, Narelle Moon, Barbora MacLean, Alexandra Hisakawa, Alma María, Rosemary Okafor, Ige Behringer, Filip Traverse, Damjan Chapman, Wulf Carlevaro, Aaron Dreschner, Kumar Dahl, Eugenio Mataracı, Ferran Simen, Xavier Hayasaka, Luis Moray, Marcos Rudaski
